src/client/Microsoft.Identity.Client/AppConfig/ManagedIdentityApplicationBuilder.cs

```diff
@@ -201,7 +140,7 @@ public IManagedIdentityApplication Build()
         /// <returns></returns>
         internal ManagedIdentityApplication BuildConcrete()
         {
-            ValidateUseOfExperimentalFeature("ManagedIdentityClient");
+            ValidateUseOfExperimentalFeature("ManagedIdentity");
```

Comment:
The GA will be part of a separate PR, I assume? (If not, I think it should be separate.)

Reply:
Yes, GA will be separate PR